armv7a9-zynq7000/syspage.c

```diff
@@ -6,7 +6,7 @@
  * Syspage
  *
  * Copyright 2020 - 2021 Phoenix Systems
```

Comment:
This is an error, should be 2020-2021 without spaces

Reply:
Corrected.